lib/model/rules.dart

```diff
@@ -0,0 +1,18 @@
+/// A class that holds game rules
+/// 
+/// The class returns general rules using static method [generalRules()]
+/// The class returns Fou-in-a-row rules using static method [fourInARowRules()]
+/// The class returns Misere rules using static method [misereRules()]
+class GameRules {
+  static String generalRules() {
```

Comment:
Let's remove postfix 'Rules' from the members, to make the code conciser.

Reply:
done